test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Common/KafkaE2EFixture.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common;
+
+/* Common fixture for all language test case classes which does -
+* Azure Infra setup and Func Apps Startup
+* Stopping Func Apps and Azure Infra cleanup
+*/
+public class KafkaE2EFixture : IAsyncLifetime
+{
+	private readonly ILogger _logger = TestLogger.GetTestLogger();
+	private AppType _appType;
+	private BrokerType _brokerType;
+	private Language _language;
+	protected bool isInitialized;
+
+	public KafkaE2EFixture()
+	{
+		_logger.LogInformation($"Kafkae2efixture for {_language} {_appType} {_brokerType}");
+	}
+
+	async Task IAsyncLifetime.DisposeAsync()
+	{
+		//Stopping Func Apps and Azure Infra cleanup
+		var testSuitCleaner = new TestSuiteCleaner();
+		await testSuitCleaner.CleanupTestSuiteAsync(_language, _brokerType);
+		_logger.LogInformation("DisposeAsync");
```

Comment:
Since this is a class which is called for different context. It is good to put context in the log (eg. Language,broker type,etc.). It will make debugging easy.

Reply:
Sure. We are planning to have a separate task for logging improvement. Will take it up as a part of that.